control_plane/src/local_env.rs

```diff
@@ -57,6 +58,11 @@ pub struct LocalEnv {
 
     #[serde(default)]
     pub safekeepers: Vec<SafekeeperConf>,
+
+    /// Every tenant has a first timeline created for it, currently the only one ancestor-less for this tenant.
+    /// It is used as a default timeline for branching, if no ancestor timeline is specified.
+    #[serde(default)]
+    pub initial_timelines: HashMap<ZTenantId, ZTimelineId>,
```

Comment:
Yet now we use `initial_timeline` concept, since pg nodes would require something to start on.
Before, `node_name`, `branch_name` was usually matching in our python tests, and we've got the timeline_id  in any of the `zenith pg ..` methods via `parse_point_in_time`  that called pageserver's `branch_get_by_name` method.
(see https://github.com/zenithdb/zenith/pull/1286/files#r807832063 and #TODO kb for more details)

Now we're not guessing branch names anymore, so have to explicitly set the timeline id into pg node.
To mitigate the verboseness in tests, the `initial_timeline` was introduced into Python test.